src/BruteForce.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+public class BruteForce extends Decipher {
+    public Map<String, Key> getKeys() {
+        Map<String, Key> mapKeys = new TreeMap<>();
+        mapKeys.put("~~~", new KeyShift3(3));
```

Comment:
~~~
~~~~~
~~~~~~~~ make it constants

Reply:
already removed the constructor with the parameter